tests/tests_bond_nm.yml

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
The test can also run with initiscripts since bond is supported there, too. Running `tests/ensure_provider_tests.py generate` will create the shims for this.

Reply:
will cover this later